packages/zowe-explorer/src/dataset/actions.ts

```diff
@@ -974,10 +981,28 @@ export async function submitMember(node: api.IZoweTreeNode) {
             { title: "Submit" }
         );
 
-        if (selection == null || selection?.title !== "Submit") {
-            return;
+        return selection != null && selection?.title === "Submit";
+    };
+
+    if (confirmationOption !== "Disabled") {
+        const ownsJob = datasetName.split(".")[0] === nodeProfile.profile?.user?.toUpperCase();
+        if (confirmationOption === "All jobs") {
```

Comment:
Is is possible to use constants for these options?
and should they be localized?

Reply:
Yep! Forgot about localization here 😅